### DISCUSSION TITLE
[RELEASE] Small House

### DISCUSSION THREAD

**Discussion by Bobthebob5** [nzp-team/nzportable#750] [Map Releases]
Just as the title suggests its a small house. 
Its a small house, they are trespassing.
My first map so it doesnt contain anything special in terms of easter eggs or anything, more of a test to learn trenchbroom.

![smallhousemap](https://github.com/nzp-team/nzportable/assets/167271269/9ecdc0ad-c0e8-4d12-ba57-945437a0ae54)

[DOWNLOAD](https://github.com/nzp-team/nzportable/files/15462612/Small.House.zip)

Its a small map and was tested on:
New 3ds
psp slim (as well as 32 and 64 ppsspp)
pc

**Comment by totallypez** [comment]
plays very nicely on nintendo switch !!! ^^

**Comment by Official-KEX** [comment]
This map is **evil**. I love it!

**Comment by MotoLegacy** [comment]
Hey there! This does not meet install standard as the download is not hosted on archive.org

**Comment by nonMat** [comment]
Fun!!! Altough having the ppsh on the wall makes this a bit too easyto some regard but it's so bad as the map is quite small. You can also get outside the map by jumping through the windows.

<img width="1920" height="1080" alt="Small_House" src="https://github.com/user-attachments/assets/e54f23dd-3881-4306-bf63-2225ac969da0" />
